### PR TITLE
Added missing fields to google.visualization TableOptions

### PR DESCRIPTION
**Pull request by dancingplatypus**
Added missing fields to google.visualization TableOptions

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://developers.google.com/chart/interactive/docs/gallery/table
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header. (These options have been present since 2016)
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

File: types/google.visualization/index.d.ts [hunk 1]
```diff
@@ -1112,9 +1112,11 @@ declare namespace google {
             alternatingRowStyle?: boolean;
             cssClassNames?: CssClassNames;
             firstRowNumber?: number;
+            frozenColumns?: number;
             height?: string;
             page?: string;
             pageSize?: number;
+            pagingButtons?: number | 'both' | 'prev' | 'next';
```
Comment (shrujalshah28) [inline types/google.visualization/index.d.ts]: Please add `auto` as a possible value for `pagingButtons`.
Reference: [Docs](https://developers.google.com/chart/interactive/docs/gallery/table#configuration-options)

Comment (shrujalshah28) [inline types/google.visualization/index.d.ts]: Please add `auto` as a possible value for `pagingButtons`.
Reference: [Docs](https://developers.google.com/chart/interactive/docs/gallery/table#configuration-options)

**Review by shrujalshah28 — CHANGES_REQUESTED** [review]

**Comment by dancingplatypus** [comment]
Done.  I don't know how I missed that.

File: types/google.visualization/index.d.ts (hunk 1 above)
Comment (dancingplatypus) [reply types/google.visualization/index.d.ts]: Thank you.  Fixed!

**Review by shrujalshah28 — APPROVED** [review]
LGTM

**Comment by uniqueiniquity** [comment]
Congratulations on your first DefinitelyTyped contribution!
Thank you for being a part of the community!

**Pull request merged**